src/prepared_statement.rs

```diff
@@ -0,0 +1,174 @@
+
+use frame::{Frame, Flag};
+use IntoBytes;
+use frame::parser::parse_frame;
+use authenticators::Authenticator;
+use error;
+use transport::CDRSTransport;
+use consistency::Consistency;
+use client::Session;
+use query::QueryParamsBuilder;
+use types::value::Value;
+use frame::frame_response::ResponseBody;
+use std::collections::BTreeMap;
+use std::convert::Into;
+use byteorder::{BigEndian, WriteBytesExt};
+
+#[derive(Debug)]
+pub struct PreparedStatement {
```

Comment:
Agreed but how often people do that? Often it is single time use. That is the reason why this would be an alternative approach, whomsoever wants more control can use the original method